ovos_dinkum_listener/voice_loop/hotwords.py

```diff
@@ -204,11 +204,13 @@ def found(self) -> Optional[str]:
         if self.state == HotwordState.LISTEN:
             engines = self.listen_words
             if not engines:
-                LOG.debug("No listen_words loaded")
+                raise RuntimeWarning(
+                    f"Waiting for listen_words but none are Available!")
         elif self.state == HotwordState.WAKEUP:
             engines = self.wakeup_words
             if not engines:
-                LOG.debug("No wakeup_words loaded")
+                raise RuntimeWarning(
+                    f"Waiting for wakeup_words but none are Available!")
```

Comment:
i think this should be a warning, since wake up could be done via bus event. as part of home automation for example